ntbs-service/wwwroot/css/_colors.scss

```diff
@@ -1,9 +1,17 @@
 $phe-red: #7f0029;
 $phe-red-focus: #610925;
-$ntbs-draft: #DCC2C6;
 $dull-red: #D8D8D87f;
 $secondary-button: #425563;
 $secondary-button-focus: #384854;
 $button-drop-shadow: #003317;
 $grey: #999999;
-$lighter-grey: #CCCCCC;
\ No newline at end of file
+$lighter-grey: #CCCCCC;
+
+$banner-header--draft: #dcc2c6;
+$banner-border--draft: #8c3a47;
+$banner-header--notified: $phe-red;
+$banner-border--notified: $phe-red;

```

Comment:
Does the border not want to be slightly different, like on the other types?

Reply:
Not according to Figma, and it seems to look decent enough with the matching border

Reply:
hmm yeah, fair enough! ;P